uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar.component.js

```diff
@@ -346,14 +426,11 @@ define([
      *
      * @return {Promise}
      */
-    function waitUntilMonthsAre (status) {
+    function waitUntilMonthIs (status) {
```

Comment:
this function is missing the status parameter doc.

Reply:
Done.